doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
 
     ```sql
-    CREATE MATERIALIZED VIEW counter AS
-        SELECT COUNT(*) FROM recentchanges;
+    CREATE TABLE symbols (
+        symbol text,
+        ticker text
+    );
+
+    INSERT INTO symbols
+    SELECT * 
+    FROM (VALUES ('Apple','AAPL'), 
+                 ('Google','GOOG'), 
+                 ('Elerium','ELER'), 
+                 ('Bespin Gas','BGAS'),
+                 ('Linen Cloth','LCLO')
+    );
+
     ```
 
-1. However,  we can also see more interesting things from our stream. For instance, who are making the most changes to Wikipedia?
+    **Note:** We are using a table for convenience, to avoid adding complexity to the guide. It's [unlikely](/sql/create-table/#when-to-use-a-table) that you'll need to use tables in real-world scenarios.
+
+1. Now, we can enrich our aggregated data with the ticker for each stock using a regular `JOIN`:
 
     ```sql
-    CREATE MATERIALIZED VIEW useredits AS
-        SELECT user, count(*) FROM recentchanges GROUP BY user;
+    CREATE MATERIALIZED VIEW cnt_ticker AS
+    SELECT s.ticker AS ticker, 
+           COUNT(*) AS cnt
+    FROM market_orders m
+    JOIN symbols s ON m.symbol = s.symbol
+    GROUP BY s.ticker;
     ```
 
+    To see the results:
+
     ```sql
-    SELECT * FROM useredits ORDER BY count DESC;
+    SELECT * FROM cnt_ticker;
+     ticker | cnt
+    --------+-----
+     AAPL   |  42
+     BGAS   |  49
+     ELER   |  68
+     GOOG   |  51
+     LCLO   |  70
     ```
+{{< /expand >}}
 
-1. If this is a factoid we often want to know, we could also create a view of just the top 10 editors we've seen.
+{{< expand "Temporal Filters" >}}
+
+A common requirement in streaming use cases is time-windowing, for example to model business processes or simply to bound resource usage. In Materialize, you can express this pattern using [temporal filters](/guides/temporal-filters/).
+
+1. If instead of computing and maintaining the _overall_ count we want to get the _moving_ count over the past minute:
 
     ```sql
-    CREATE MATERIALIZED VIEW top10 AS
-        SELECT * FROM useredits ORDER BY count DESC LIMIT 10;
+    CREATE MATERIALIZED VIEW cnt_sliding AS
+    SELECT symbol, 
+           COUNT(*) AS cnt
+    FROM market_orders m
+    WHERE EXTRACT(EPOCH FROM (ts + INTERVAL '1 minute'))::bigint * 1000 > mz_logical_timestamp()
+    GROUP BY symbol;
     ```
 
-    We can then quickly get the answer to who the top 10 editors are:
+1. To see the results:
 
     ```sql
-    SELECT * FROM top10 ORDER BY count DESC;
+    SELECT * FROM cnt_sliding;
+
+       symbol    | cnt
+    -------------+-----
+     Apple       |  31
+     Google      |  40
+     Elerium     |  46
+     Bespin Gas  |  35
+     Linen Cloth |  45
     ```
 
-Naturally, there are many interesting views of this data. If you're interested
-in continuing to explore it, you can check out the stream's documentation from
-Wikipedia.
-
-Once you're done, don't forget to stop `curl` and `rm wikirecent`.
+{{< /expand >}}
```

Comment:
For temporal filters, we should tell people to wait a few minutes and retrieve results again, so they can see the results changing over time (the sliding of the window).

Reply:
I'll add some more context, it does end quite abruptly!